webpack/src/Extends/Host/PuppetTab/SubTabs/Reports/components/DescriptionCard.js

```diff
@@ -1,7 +1,11 @@
 import PropTypes from 'prop-types';
-import React from 'react';
-import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
```

Comment:
Imho it would be better to use `CardTemplate` from core as it is,
but protect plugins in case the [Expansion Context](https://github.com/theforeman/foreman/blob/2c252511913090d3defe58cbc863af5c5920811d/webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js#L27) is `undefined`

Reply:
I disagree, because cardtemplate needs `dispatch` from context in order to expand, and while its good that the page doesn't crash with the foreman pr the card is not expandable